### ISSUE TITLE
Detect junk input

### ISSUE THREAD

**Issue by dabreegster** [acteng/atip#65]
https://github.com/acteng/atip/blob/5ff0942dc3790cda6c5cc302ae0314447be8e221/components/EntireScheme.svelte#L66

You can load an arbitrary file. If it's not JSON, it'll fail reasonably. But otherwise, we don't even constrain the input to be GeoJSON. There's many more things we should require (no CRS on the geojson, no unexpected nested feature collections). We should probably just ignore unknown or invalid properties, but unsure. (Longer term we could come up with a proper schema for the data we collect, with a strategy for backwards compatibility)

**Comment by Robinlovelace** [comment]
Trying this in session with @sparrow0hawk. Test input:

```
echo 'junk-input' > test-junk.txt
```

**Comment by Robinlovelace** [comment]
Current behaviour is really not bad: detects that it's not valid json.

![image](https://user-images.githubusercontent.com/1825120/212329219-cb3d47d1-8af7-436c-99be-2273fe691e3c.png)

**Comment by Robinlovelace** [comment]
Ha, OK wasn't expecting this!

![image](https://user-images.githubusercontent.com/1825120/212331691-d6a8954b-048f-46c3-861d-71559caa8325.png)

**Comment by Sparrow0hawk** [comment]
Tested with non ATIP geojson ([source](https://raw.githubusercontent.com/Sparrow0hawk/mynearestleedsmarginal/main/assets/data/leeds_wards_2018.geojson))

**Comment by Robinlovelace** [comment]
I see this as a low priority. Key question: how to detect non ATIP input?

**Comment by Sparrow0hawk** [comment]
Could adjust the schema for ATIP-generated geojson files to include a high level field specifying ATIP-generated. Could then validate this within `loadFile`.

**Comment by dabreegster** [comment]
While testing this, I thought of another mistake people could make: loading a file from a different authority. We could check for this and ban it, but maybe there are valid reasons why someone would want to load an adjacent scheme? Or we could prompt them to go load the scheme in the other page?

**Comment by Robinlovelace** [comment]
I don't think we should put up too many guardrails. There may well be valid reasons: looking at what the neighbours have planned is a good way to up you game! No harm in a prompt but the fact that the map zooms to a new place will surely be a prompt in itself?

**Comment by dabreegster** [comment]
Flagging https://zod.dev/ as possibly interesting for this issue and maybe our schemas/forms
